sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -19,7 +19,7 @@
 
 == Notice
 
-Copyright (c) 2022 Intel Corporation.  All rights reserved.
+Copyright (c) 2023 Intel Corporation.  All rights reserved.
```

Comment:
I think @gmlueck asked to change the copyright to "2022 - 2023".

Reply:
good spot, fixed